### DISCUSSION TITLE
I would love to have feature to schedule and backup my database other than a s3 bucket.

### DISCUSSION THREAD

**Discussion by Rounak826** [Dokploy/dokploy#88] [Ideas]
- Maybe allowing some custom script
- to save backup in Google Drive or other cloud storage
- to download the backup and later restore it from there

**Comment by Siumauricio** [comment]
What type of script would you like to run? we can make some brainstorming on how we can implement that feature

**Reply by Rounak826** [reply]
A Cron Job Script for Generating Dump files of my Database and using a library like gdrive3 to upload the dump file to my Google Drive.

**Reply by Rounak826** [reply]
as @ChispasHK suggested rsync or rclone. That would also work for me I have also come across the name Restic. I am open to any alternative allowing me to back up my volumes other than s3.

**Comment by ChispasHK** [comment]
What do you think about the idea of implementing **rsync** or **rclone** for our backup task?

**Reply by Rounak826** [reply]
yes, it sounds good to me
